CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contributing Guidelines
+
+The KUDO Operators project accepts contributions via GitHub pull requests. This document outlines the process to help get your contribution accepted.
+
+## Sign the CLA
+
+Kubernetes projects require that you sign a Contributor License Agreement (CLA) before we can accept your pull requests.
```

Comment:
Why does that apply to KUDO?

Reply:
@mattj-io can elaborate more but this is part of the preparation we need to become a CNCF project

Reply:
Because the whole project has been submitted as CNCF sandbox. We need to follow the same governance model.

Reply:
moved to DCO, as thats where KUDO is moving now
https://github.com/kudobuilder/kudo/pull/1164

Reply:
There's more discussions needed here, we're not sure what applies to KUDO right now.